docs/operators/how-to/configure.md

```diff
@@ -1,12 +1,135 @@
 # Configure a nwaku node
 
 Nwaku can be configured to serve the adaptive needs of different operators.
-This page serves as an index of tutorials explaining how to configure your nwaku node for different use cases.
+
+> :bulb: **Tip:** The recommended configuration method is through environment variables.
+
+## Node configuration methods
+
+One node can be configured using a combination of the following methods:
+
+1. Command line options and flags
+2. Environment variables
+3. Configuration file (currently, only TOML format is supported)
+4. Default value
+
+Note the precedence order, each item takes precedence over the item below it.
```

Comment:
I had to read this couple of times `precedence over the item below it`. Perhaps be more explicit? "each item overrides the one below."

Reply:
How would you rephrase it? What do you suggest?

Reply:
> "each item overrides the one below."

Note the precedence order, where "each item overrides the one below."